### ISSUE TITLE
Not reading main tmux.conf 

### ISSUE THREAD

**Issue by naturedamends** [gpakosz/.tmux#645]
this is ran inside a docker container, ubuntu:latest. i've seen it work, not sure what's happened.

Its not working even if I link tmux.conf to ~/.tmux.conf

```
 tmux -V
tmux 3.2a
```

setup script:
```
set -e

OM_TMUX="$HOME/.oh-my-tmux"
git clone https://github.com/gpakosz/.tmux.git "$OM_TMUX"

TMUX_CONF_FOLDER="$HOME/.config/tmux"
TMUX_MAIN_CONF="$TMUX_CONF_FOLDER/tmux.conf"

# install gloabl my-oh-tmux tmux conf
mkdir -p "$TMUX_CONF_FOLDER"
ln -s "$OM_TMUX/.tmux.conf" "$TMUX_MAIN_CONF"

# install user config
TMUX_LOCATION="$TMUX_MAIN_CONF.local"
[ -L "$TMUX_LOCATION" ] && rm "$TMUX_LOCATION"
ln -s "$DOTFILES_SHARED/.tmux.conf" "$TMUX_LOCATION"
```

```
ls -la ~/.config/tmux
total 12
drwxr-xr-x 2 natured natured 4096 May  8 08:58 .
drwxr-xr-x 1 natured natured 4096 May  8 09:01 ..
lrwxrwxrwx 1 natured natured   36 May  8 08:58 tmux.conf -> /home/natured/.oh-my-tmux/.tmux.conf
lrwxrwxrwx 1 natured natured   59 May  8 08:58 tmux.conf.local -> /home/natured/repos/dotfiles/machines/env/shared/.tmux.conf
```
```
ls -la
total 320
drwxr-x--- 1 natured natured   4096 May  8 09:49 .
drwxr-xr-x 1 root    root      4096 May  7 17:49 ..
-rw-r--r-- 1 natured natured    220 Jan  6  2022 .bash_logout
-rw-r--r-- 1 natured natured   3823 May  7 07:16 .bashrc
drwxr-xr-x 7 natured natured   4096 May  8 09:03 .cache
drwxr-xr-x 1 natured natured   4096 May  8 09:01 .config
drwxr-xr-x 4 natured natured   4096 May  8 09:01 .dotnet
drwxr-x--- 1 natured natured   4096 May  8 09:02 .local
drwxr-xr-x 4 natured natured   4096 May  8 08:58 .npm
drwxr-xr-x 3 natured natured   4096 May  8 09:01 .nuget
drwxr-xr-x 4 natured natured   4096 May  8 08:58 .oh-my-tmux
drwxr-xr-x 1 natured natured   4096 May  8 08:58 .oh-my-zsh
-rw-r--r-- 1 natured natured    807 Jan  6  2022 .profile
drwxr-xr-x 1 root    root      4096 May  8 08:57 repos
drwxr-xr-x 3 natured natured   4096 May  8 08:58 .ssh
-rw-r--r-- 1 natured natured      0 May  8 08:58 .sudo_as_admin_successful
drwxr-xr-x 3 natured natured   4096 May  8 09:02 .vim
drwxr-xr-x 3 natured natured   4096 May  8 09:03 .yarn
-rw-r--r-- 1 natured natured  49213 May  8 09:41 .zcompdump
-rw-r--r-- 1 natured natured  51406 May  8 09:41 .zcompdump-63131fbcbc05-5.8.1
-r--r--r-- 1 natured natured 117560 May  8 09:41 .zcompdump-63131fbcbc05-5.8.1.zwc
lrwxrwxrwx 1 natured natured     58 May  8 08:58 .zshenv -> /home/natured/repos/dotfiles/machines/env/dev/variables.sh
-rw------- 1 natured natured    364 May  8 09:49 .zsh_history
-rw-r--r-- 1 natured natured    318 May  8 08:58 .zshrc
```

So i'm just left with the default
![image](https://user-images.githubusercontent.com/120284608/236793905-b9f5953e-36df-4eb2-9abb-43217101aed7.png)

I can force it to use the config by passing it path as a argument via the `tumx -f` flag. So it works fine.

If i source-file I get the following
![image](https://user-images.githubusercontent.com/120284608/236845227-64add6f8-e1e9-4e8c-96a7-022796c34238.png)

I have added to the user conf section of the .local
```
# start with mouse mode enabled
set -g mouse on
set -g default-shell /usr/bin/zsh
```

**Comment by gpakosz** [comment]
BTW what's this `_zsh_tmux_plugin_run` window title?

**Comment by naturedamends** [comment]
> Then follow the manual installation instructions listed in
I've tried my best

> Make sure you have no tmux procress running
I can confirm there was no existing sessions.

> BTW what's this `_zsh_tmux_plugin_run` window title?

Not sure, it may be related to Iterm Tmux intergration

https://iterm2.com/documentation-tmux-integration.html

**Comment by naturedamends** [comment]
> BTW what's this `_zsh_tmux_plugin_run` window title?

Running 
```
command tmux
```
Produces the expected output. 🥇 thanks. Not sure if this is upstream. But feel free to close 👍

**Comment by deepanchal** [comment]
Setting `$ZSH_TMUX_CONFIG` in `.zshrc` doesn't work. You will have to set this env var in your `.zshenv` file. See: https://zsh.sourceforge.io/Intro/intro_3.html?ref=zerotohero.dev

```bash
# ZSH Config Vars
export ZSH_TMUX_CONFIG="$XDG_CONFIG_HOME/tmux/tmux.conf"
```

**Comment by bubanoid** [comment]
I faced with a very similar problem. 

    tmux -V
    tmux 3.3a

I have `zsh` as a default shell, `oh-my-zsh` is installed too, but without `tmux` plugin. I followed the instructions and installed `oh-my-tmux` in the `~/.config/tmux` folder. Unfortunately, it doesn't work out of the box. Isn't there missing some steps in the instructions?

It looks like it is necessary to create a soft link for the `~/.tmux.conf` file or set `TMUX_CONF` and `TMUX_CONF_LOCAL` to the `~/.config/tmux/tmux.conf` and `~/.config/tmux/tmux.conf.local` correspondently.

**Comment by naturedamends** [comment]
> I faced a very similar problem. 
> 
> 
> 
>     tmux -V
> 
>     tmux 3.3a
> 
> 
> 
> I have `zsh` as a default shell, `oh-my-zsh` is installed too, but without `tmux` plugin. I followed the instructions and installed `oh-my-tmux` in the `~/.config/tmux` folder. Unfortunately, it doesn't work out of the box. Isn't there missing some steps in the instructions?
> 
> 
> 
> It looks like it is necessary to create a soft link for the `~/.tmux.conf` file or set `TMUX_CONF` and `TMUX_CONF_LOCAL` to the `~/.config/tmux/tmux.conf` and `~/.config/tmux/tmux.conf.local` correspondently.

Tell zsh about your config location? https://github.com/gpakosz/.tmux/pull/648

**Comment by bubanoid** [comment]
I identified the root cause of my issue. I created an empty `~/.tmux.conf` and missed that completely.  When attempting to apply the correct tmux configuration file located in any other than the home directory it has no effect. This was due to the `TMUX_CONF` environment variable being set to the existing `~/.tmux.conf` file. So, I needed to remove the `~/.tmux.conf` file first.

Sorry for bothering you.
